static/js/client.js

```diff
@@ -124,6 +149,10 @@ var app = {}, browserSettings = {}, browserStorage = $.localStorage;
         bg_title = s(scaleBg(currentMgdl)) + s(deltaDisplay) + s(direction.info(latestSGV).label) + bg_title;
       }
     }
+    } else {
+    	bg_title = message.title + ' ' + message.message;
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['bg_title' used out of scope.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/769371567/issues/source?bid=2452379&fileBranchId=2457543#l153)